examples/demo4.yaml

```diff
@@ -103,14 +106,32 @@ gal :
     type : Sum
 
     items :
+        # 0.6 of flux is in the total disk.  Further divide this
+        # into a smooth disk and knots of star formation, modeled
+        # as point sources distributed via a random walk
+        #
+        # note the knots get the same hlr as the smooth disk, and
+        # get sheared by the same ellipticity, but the knots will
+        # themselves have additional shape noise
+
         - 
             type : Exponential
             half_light_radius : { type : Catalog , col : 5 }
             ellip : 
                 type : E1E2 
                 e1 : { type : Catalog , col : 6 }  
                 e2 : { type : Catalog , col : 7 }
-            flux : 0.6
+            flux : 0.18
```

Comment:
I think at this (early) stage in the demo progression, leaving these hard-coded is appropriate, although this demo already has the `some_variables` bit at the top, so you could put bulge_frac and knot_frac at the top if you want.

You could also intentionally omit of the last component flux and point out that when the overall galaxy has a flux specified (as it is here), the fluxes of each component are the fractional flux, and the final component's flux fraction will be automatically set to make the total = 1.